<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myDialog</class>
 <widget class="QDialog" name="myDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>257</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#myDialog {
    font-family:'Roboto', sans-serif;
    border-radius: 5px;
    background-color: #1a2030;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>21</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>43</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1a2030;
font-weight:350;
</string>
     </property>
     <property name="text">
      <string>提示</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="l1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 输入框的布局和颜色，以及它的hover状态 */
QLineEdit {
    border-radius: 6px;
    padding: 8px 12px;
    padding-top: 8px;
    padding-right: 12px;
    padding-bottom: 8px;
    padding-left: 12px;
    color: #34495e;

}
QLineEdit:focus{
    border: 2px solid #bdc3c7;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入文章标题</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;border-width:0;border-style:outset;
background-color:  #1a2030;
</string>
     </property>
     <property name="text">
      <string>这是一段信息</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="l2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 输入框的布局和颜色，以及它的hover状态 */
QLineEdit {
    border-radius: 6px;
    padding: 8px 12px;
    padding-top: 8px;
    padding-right: 12px;
    padding-bottom: 8px;
    padding-left: 12px;
    color: #34495e;

}
QLineEdit:focus{
    border: 2px solid #bdc3c7;
}</string>
     </property>
     <property name="placeholderText">
      <string>输入已有的分类编号或新建分类</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="t2">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#t2{
font-weight: 550;
color:white
}</string>
     </property>
     <property name="text">
      <string>文章分类：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="t1">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#t1{
font-weight: 550;
color:white
}</string>
     </property>
     <property name="text">
      <string>文章标题：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton#bt_ok{
    border: 1px solid rgb(64, 158, 255);
    font-size: 16px;
    font-weight: normal;
    min-width: 50px;
    width:50;
    line-height: 1.4;
    min-height: 28;
    border-radius: 5px;
    padding: 5px 7px;
    color: white;
    background-color: #3498db;
}
QPushButton#bt_ok:hover {

    background-color: #5dade2;
}
 QPushButton#bt_ok:pressed {
    border: 1px solid #138496;
    background-color:#2c81ba;
}

QPushButton#bt_no{
    border: 1px solid rgb(220, 223, 230);
    font-size: 16px;
    font-weight: normal;
    min-width: 50px;
    width:50;
    line-height: 1.4;
    min-height: 28;
    border-radius: 5px;
    padding: 5px 7px;
    color: rgb(96, 98, 102);
    background-color: white;
}
 QPushButton#bt_no:hover {
    background-color: #ecf5ff;
	color:#3498db;
}
QPushButton#bt_no:pressed {
       border: 1px solid #3498db;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bt_no">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bt_ok">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
